variables.tf

```diff
@@ -98,6 +98,21 @@ variable "monitor_iam_access" {
   }
 }
 
+variable "sns_endpoint" {
+  type        = string
+  description = "Endpoint for SNS topic subscription"
+}
+
+variable "sns_endpoint_protocol" {
+  type        = string
+  description = "Endpoint protocol for SNS topic subscription"
+}
+
+variable "sns_security_topic_subscription" {
+  type        = bool
+  description = "Enable SNS aggregated security topic subscription"
+}
```

Comment:
Could these values be specified in a single `sns_security_subscription` object?

Reply:
done